### PR TITLE
Numeric: add bounds

### PR DESCRIPTION
**Pull request by sterlinghirsh**
This is connected to https://github.com/iFixit/ifixit/pull/16871

### CODE REVIEW

File: assets/js/templates/popover/popover-checkbox.html [hunk 1]
```diff
@@ -1,4 +1,7 @@
 <label class="checkbox control-group">
-  <input type="checkbox" data-type="<%= type %>" class="input-inline field" name="<%= name %>" id="<%= name %>"<% if(value) {%> checked="checked" <% } %>>
+  <input type="checkbox" data-type="<%= type %>" class="input-inline field"
+   name="<%= name %>" id="<%= name %>"<% if(value) {%> checked="checked" <% } %>
+   <% if (typeof auxiliary !== 'undefined' && auxiliary) { %> data-auxiliary="auxiliary" <% } %>
+   >
```
Comment (davidrans) [inline assets/js/templates/popover/popover-checkbox.html:5]: Why did this need to be added to the checkbox popover also?

**Comment by davidrans** [comment]
CR ☕️

**Comment by sterlinghirsh** [comment]
Checkboxes are set of the aux attributes like limit decimals and limit min

**Comment by davidrans** [comment]
CR ☕️

**Pull request merged**